test/rules/avoid_defining_a_class_that_contains_only_static_members.dart

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) 2017, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+// test w/ `pub run test -N avoid_defining_a_class_that_contains_only_static_members`
+
+class Bad { // LINT
+  static int a;
+
+  static foo() {}
+}
+
+class Bad2 extends Good1 { // LINT
```

Comment:
Yes, but every class extends Object and it has nonstatic members (like toString), so I thinks the spirit of the rule, is not to create classes that locally only have static members.